content/articles/getting-started-clickfunnels.markdown

```diff
@@ -0,0 +1,222 @@
+1. Now that you have accepted the push of the domain, you need to change the Registrant information. This information states that you are the owner of the domain. When the domain was with ClickFunnels, they were listed as the owner, which is why we need to update this. See [Updating the registrant information](#updating-the-registrant-information) for a description of each field.
+
+    <note>
+    After the domain is pushed into your DNSimple account, you must also change the ownership of the domain from ClickFunnels to yourself. This [locks the domain for further transfers](/articles/icann-60-day-lock-registrant-change/) for 60 days in accordance with ICANN's rules, meaning that you have to keep your domain here at DNSimple for 60 days before you can transfer the domain elsewhere. Hopefully, you aren't looking to move it but if you are, we will discuss some options later in the article.
+    </note>
+
+    <warning>
+    You will receive a verification email to the email address you used for the domain contact requesting you verify you made a change to the contact information. If you are moving your domain from ClickFunnels, you did make this change and you will want to verify it as soon as you receive the email. Sometimes it lands in spam, so be on the lookout for it.
+
+    Ignoring this message will lead to a suspended website, meaning it won't work. If this does happen, as soon as you verify it the suspension will be lifted.
+
+    You will receive a confirmation email after you have verified letting you know the verification was successful. There is nothing you need to do with this email.
+    </warning>
+
+1. Click <label>Accept Push</label>. You will then be brought to the domain's page and you'll see a banner stating the domain has been successfully pushed to your account.
+</div>
+
+Congratulations! You're now the owner of the domain and are managing the DNS.
+
+### Other options when getting started with a new account
+
+#### Add a domain
+
+If you were purchasing a new domain or transferring a domain in from another provider, like GoDaddy, you would choose this. You are not transferring a domain in from ClickFunnels because they use DNSimple as their registrar already. You are moving your domain from ClickFunnels' DNSimple account into your DNSimple account.
+
+#### Request an SSL certificate
+
+If you need to purchase a certificate right away, you can choose this option. ClickFunnels includes a certificate to make your site secure with their services. We will go over Certificates shortly, this option isn't necessary right now.
+
+#### Explore the API
+
+The API is a great resource for those with a technical background to take advantage of automation. It may be something that you want to look into and explore. I will provide links to helpful articles explaining the API at the end of this document, you don't need to dive in just yet.
+
+### Updating the registrant information
```

Comment:
Hold on, I'm confused. Why is this down here?

Reply:
@SarahCaminiti1 i put it here because the table wouldn't render properly when included in the `DIV`.